cli/test_cmd.go

```diff
@@ -114,6 +114,16 @@ func (c *TestCmd) Execute(args []string) error {
 			}
 			expectedDir := filepath.Join(tree, "../../expected", exeMethod, filepath.Base(tree))
 			actualDir := filepath.Join(tree, "../../actual", exeMethod, filepath.Base(tree))
+
+			_, err := os.Stat(expectedDir)
+			if err != nil && os.IsNotExist(err) {
+				// (alexsaveliev) we may have some tests available only for docker or program mode
```

Comment:
Why would a test only exist for docker or program mode and not both? The two should have equivalent behavior (as much as practically possible).